src/wstore/ordering/tests/tests.py

```diff
@@ -299,6 +299,7 @@ def _missing_postal(self):
 
     @parameterized.expand([
         ('basic_add', BASIC_ORDER, BASIC_PRICING, _basic_add_checker),
+        ('basic_add_terms_not_accepted', BASIC_ORDER, BASIC_PRICING, None, None, 'OrderingError: You must accept the terms and conditions of the offering to acquire it', False),
```

Comment:
I think there is  missing test case, the product do not have terms and conditions and the terms_accepted flag is false, the order processing should be success